Rlisa33.md

```diff
@@ -0,0 +1,20 @@
+# **Enjoying Github Intern Software Delevopment Service!**
+[Rlisa33 Homepage](https://github.com/Rlisa33/Rlisa33.github.io)
+- (win_20170328_20_18_28_pro)
+- GitHub Intern Experience!
+- Rachel Mathurin
+- Eastern Time Florida
+- OS Window 7
+- I am an online student at the University of Phoenix. I am studying to earn my Master of Science in Accountancy. 
+- I lived in Florida for over four years now. I like reading, and surfing the net. I like drawing. 
+- I like software engineering and computer science and math as well as accounting.
+- I am eager to get a job to start working and I believe volunteering with help my resume.
```

Comment:
Your bio should be a well formatted paragraph **or** you should make it read more like a list such as:

**Some facts about me:**

- I am an online student at the University of Phoenix studying to earn my Master of Science in Accountancy.
- I lived in Florida for over four years now.
 . . . and so on